packages/ag-charts-community/src/chart/series/polar/pieSeries.ts

```diff
@@ -1522,9 +1523,11 @@ export class PieSeries extends PolarSeries<PieNodeDatum, Sector> {
     }
 
     getDatumIdFromData(datum: any) {
-        const { calloutLabelKey, sectorLabelKey } = this;
+        const { calloutLabelKey, sectorLabelKey, legendItemKey } = this;
```

Comment:
Does this conflict/supersede with the fallback to `legendItemValue` in `getDatumId()`?

Reply:
Fair point - `legendItemValue`/`legendItemKey` needs to take precedence over the other options, and then `legendItemValue` is a fallback is redundant, so I've removed it.